smhi/smhi_lib.py

```diff
@@ -398,6 +410,7 @@ def _get_all_forecast_from_api(api_result: dict) -> OrderedDict:
             precipitation,
             wind_direction,
             wind_speed,
+            wind_speed,
```

Comment:
Is wind_speed included twice?

Reply:
It is kinda flunky design hehe but there are only one wind speed from the original api call so I enter that into the object but one of them will be turned into mean_wind_speed. I probably could design that better..

Reply:
Will not merge this until there are demand for it